### PR TITLE
Create heading to products page

### PR DESCRIPTION
**Pull request by MostafaBalousha123**
- Create a heading for the products page .
- create search field .
- add the style that will be directed to mobile devices and iPads .

### CODE REVIEW

**Review by naghamabuwarda — APPROVED** [review]

File: productsPage/products.html [hunk 1]
```diff
@@ -6,9 +6,61 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="products.css" />
     <title>Products</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+   // icon Library
```
Comment (naghamabuwarda) [inline productsPage/products.html:10]: change the comment

**Review by naghamabuwarda — CHANGES_REQUESTED** [review]

File: productsPage/products.html [hunk 2]
```diff
@@ -6,9 +6,59 @@
...
+              type="text"
+              placeholder="    Search"
+              class="search-input"
+            />
+          </form>
+      <!-- header element for desktop-->
+      <nav >
+        <li>   <a href="/index.html"  >Home </a>  </li>
+        <li>    <a href="/cartPage/cart.html"  > My Cart</a> </li>
+        <li>    <a href="/aboutPage/about.html" >About Us </a> </li>
```
Comment (sara219) [inline productsPage/products.html:33]: the links will not work in GitHub pages with this path, you need to link it directly 

```suggestion
<li>   <a href="../index.html"  >Home </a>  </li>
        <li>    <a href="../cartPage/cart.html"  > My Cart</a> </li>
        <li>    <a href="../cartPage/cart.html" >About Us </a> </li>
```

File: productsPage/products.html [hunk 3]
```diff
@@ -6,9 +6,59 @@
...
 <body>
-    
+    <section class="continar-all-page">
+              <!--  *************** Start  Header  ************* -->
+
+        <!-- header for mobile and ipad  -->
+        <header>
+       <a href="/index.html"> <i class="ri-arrow-left-s-line"></i></a>
+          <form class="search-form">
+            <img src ="/images/Search.svg"class = "search-icon" alt="" />
```
Comment (sara219) [inline productsPage/products.html:22]:
```suggestion
          <form class="search-form">
           <img src ="../images/Search.svg"class = "search-icon" alt="" />
```

**Review by sara219 — CHANGES_REQUESTED** [review]

File: productsPage/products.html (hunk 2 above)
Comment (MostafaBalousha123) [reply productsPage/products.html]: done

**Review by sara219 — APPROVED** [review]

**Pull request merged**